internal/output/html.go

```diff
@@ -335,11 +381,89 @@ func buildHTMLResult(ecosystemMap map[string][]HTMLSourceResult, resultCount HTM
 	})
 
 	ecosystemResults = append(ecosystemResults, osResults...)
+	var layers []LayerInfo
+	if isContainerScanning {
+		layers = getAllLayers(ecosystemResults)
+	}
+	vulnTypeCount := getVulnTypeCount(ecosystemResults)
 
 	return HTMLResult{
-		EcosystemResults: ecosystemResults,
-		HTMLVulnCount:    resultCount,
+		EcosystemResults:    ecosystemResults,
+		HTMLVulnCount:       resultCount,
+		IsContainerScanning: isContainerScanning,
+		AllLayers:           layers,
+		VulnTypeCount:       vulnTypeCount,
+	}
+}
+
+func getVulnTypeCount(result []HTMLEcosystemResult) VulnTypeCount {
```

Comment:
nit: you should be able to take advantage of default values here I think, and so just do a `var vtc VulnTypCount` to get all the "variables" initialized to zero

Reply:
Thanks, updated!